remote-wallet/src/remote_wallet.rs

```diff
@@ -149,6 +161,21 @@ impl RemoteWalletManager {
             }
         }
 
+        for device in trezor_client::find_devices(false) {
```

Comment:
I'm not sure this will correctly handle multiple Trezors connected at the same time since `pubkey` is  set to `None` ... an edge case for sure, but I just had multiple hardware wallets connected two days ago, so it does happen:D

Not enough to block shipping, but a small follow-up can use e.g. `trezor.features()?.device_id` to do this.